crates/collateral-contract/script/deploy.sh

```
@@ -0,0 +1,15 @@
+export NETUID=39
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)